### PR TITLE
CDAP-11632 Reduce the size of the ProgramScheduleStoreDataset type.

### PR DESCRIPTION
**Pull request by anwar6953**
https://issues.cask.co/browse/CDAP-11632
https://builds.cask.co/browse/CDAP-RUT1012-1

### CODE REVIEW

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/Schedulers.java [hunk 1]
```diff
@@ -56,7 +56,8 @@
  * Common utility methods for scheduling.
  */
 public class Schedulers {
-  public static final String STORE_TYPE_NAME = ProgramScheduleStoreDataset.class.getName();
+  // we use the simple name for the type, due to CDAP-11632
```
Comment (sreevatsanraman) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/Schedulers.java]: Don't comment with JIRAs that will be fixed with the current PR. you can add external JIRAS or Todo JIRAS that are from CDAP project

Comment (anwar6953) [reply cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/Schedulers.java]: What do you mean by 'external JIRAs'?

Comment (sreevatsanraman) [reply cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/Schedulers.java]: Sentry, Hbase, etc.

Comment (sreevatsanraman) [reply cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/Schedulers.java]: Remove CDAP-11632

**Comment by sreevatsanraman** [comment]
👍  after the minor comment is fixed.

**Pull request merged**